KarhooUISDK/CommonUI/Layout/KarhooColors+DefaultSet.swift

```diff
@@ -62,31 +62,19 @@ public extension KarhooColors {
     // MARK: - Background Colors
 
     var background1: UIColor {
-        UIColor.get(lightModeColor: UIColor(hex: "#FAFAFA"), darkModeColor: UIColor(hex: "#232323"))
+        UIColor.get(lightModeColor: UIColor(hex: "#FAFAFA"), darkModeColor: UIColor(hex: "#121212"))
     }
 
     var background2: UIColor {
-        UIColor.get(lightModeColor: UIColor(hex: "#F2F2F2"), darkModeColor: UIColor(hex: "#252525"))
+        UIColor.get(lightModeColor: UIColor(hex: "#FFFFFF"), darkModeColor: UIColor(hex: "#232323"))
```

Comment:
Shouldn't background 1 be the lightest (in light mode) and darken as the background number increases? The other way around for dark mode.

As the colors are defined right now it looks like background 1 is darker than 2 and background 3 is darker than both 1 and 2.

Reply:
Yes, and it is how it looks in Figma: 
<img width="1411" alt="Screenshot 2022-11-15 at 15 17 15" src="https://user-images.githubusercontent.com/97114085/201941896-e7661708-1dd5-4b25-a136-b763799971a6.png">